plugin/storage/integration/integration.go

```diff
@@ -75,6 +75,10 @@ type StorageIntegration struct {
 	// Skip Archive Test if not supported by the storage backend
 	SkipArchiveTest bool
 
+	// TODO: remove this after upstream issue in OTEL jaeger translator is fixed
+	// Skip testing trace binary tags, logs, and process
+	SkipBinaryAttrs bool
```

Comment:
please book a ticket referring to this commit

Reply:
Booked it at https://github.com/jaegertracing/jaeger/issues/5341